### DISCUSSION TITLE
OPEN AI 翻譯不支持內容

### DISCUSSION THREAD

**Discussion by Topbcy** [metatube-community/jellyfin-plugin-metatube#151] [General]
使用OPEN AI並填入API來使用翻譯，電影內容欄位會顯示無法使用，不支持此內容翻譯

**Comment by xjasonlyu** [comment]
是的，这是一个known issue，因为OpenAI有相关的Content Policy。

**Reply by Topbcy** [reply]
尷尬了，這樣作者們不就白開發了嗎 .......

**Reply by xjasonlyu** [reply]
从某种意义上来说，是的。

**Reply by Topbcy** [reply]
好的，也感謝開發者作者們的辛苦，謝謝
